backend/src/main/java/com/woowacourse/levellog/feedback/application/FeedbackService.java

```diff
@@ -62,6 +62,17 @@ public FeedbacksDto findAll(final Long levellogId, final Long memberId) {
         return new FeedbacksDto(responses);
     }
 
+    public FeedbackDto findById(final Long levellogId, final Long feedbackId, final Long memberId) {
+        final Feedback feedback = getFeedback(feedbackId);
+        final Levellog levellog = getLevellog(levellogId);
+        final Member member = getMember(memberId);
+
+        validateTeamMember(levellog.getTeam(), member, "자신이 속한 팀의 피드백만 조회할 수 있습니다.");
+        validateLevellog(feedback, levellog);
```

Comment:
```suggestion
        feedback.validateLevellog(levellog);
```

굳이 메서드로 추출한 이유가 있나요? 과도한 메서드 추출이라고 생각해요 😉